requirements.in

```diff
@@ -1,28 +1,31 @@
-flask
-flask-appconfig>=0.10
-flask-security
-flask-sqlalchemy
-flask-restful
-flask-cors
-jsl
+tenacity
+Werkzeug
+Flask_Cors
+Flask_SQLAlchemy
+Flask
+flask_appconfig
+py_lru_cache
+lxml
+requests
+SQLAlchemy
+raven
+pydantic
+pytest
+Flask_Cache
+Flask_RESTful
+requests_futures
+semantic_version
 # there's a regression in mod_wsgi 4.5.4:
 # https://github.com/GrahamDumpleton/mod_wsgi/issues/147
 # install httpd-devel in order to have apxs command, otherwise build fails
 mod_wsgi!=4.5.4
-psycopg2-binary
-# in order for pyjwt to be able to use RS256 we need pycrypto
-cryptography
-pyjwt>=1.6.1
-requests-futures
-Flask-Cache
-semantic_version
-py_lru_cache
-radon==3.0.1
-pygithub
-codecov
-lxml
-gitpython
-raven[flask]
+flask
+flask-appconfig>=0.10
 pydantic
-tenacity
-semver
\ No newline at end of file
+blinker
+pygithub
+psycopg2-binary
+git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git#egg=F8A_WORKER_VERSION
```

Comment:
Format is not valid, we need to have egg=<folder Path> and before that we need to tag this deps with version?
Same goes for other three deps below.

Example Syntax:
https://github.com/fabric8-analytics/fabric8-analytics-version-comparator.git@8a57ac7#egg=f8a_version_comparator

Reply:
@dgpatelgit After adding versions(commit hash) for the other three deps the docker build fails.

Reply:
It is failing because the mentioned eggs are incorrect. Please check the suggestion.